modules/kmm-gathering-data-for-kmm-hub.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-gathering-data-for-kmm-hub_{context}"]
+= Gathering data for KMM-Hub
+
+.Procedure
+
+. Gather the data for the KMM operator hub controller manager:
++
+[source,terminal]
+----
+export MUST_GATHER_IMAGE=$(oc get deployment -n openshift-kmm-hub kmm-operator-hub-controller-manager -ojsonpath='{.spec.template.spec.containers[?(@.name=="manager")].env[?(@.name=="RELATED_IMAGES_MUST_GATHER")].value}')
+----
++
+[source,terminal]
+----
+$ oc adm must-gather --image="${MUST_GATHER_IMAGE}" -- /usr/bin/gather -u
+----
```

Comment:
Ideally, these two commands (here and in the equivalent KMM Operator proc's module) should be separate steps or substeps. Maybe like:

```
. Gather the data for the KMM-Hub Operator controller manager:

.. Set the `MUST_GATHER_IMAGE` variable:
+
[source,terminal]
----
$ export MUST_GATHER_IMAGE= [...]
----

.. Run the `must-gather` tool:
+ 
[source,terminal]
----
$ oc adm must-gather [...]
----
```